test/tests/functional/pbs_basil_support.py

```diff
@@ -326,9 +326,12 @@ def test_hbmemm_rsc(self):
             self.logger.info("KNL vnode list: %s" % (knl_vnodes))
 
         # Add 'hbmem' to the resources line in sched_config.
-        self.scheduler.add_resource('hbmem')
+        # For actual cray platform post_install will add this resource
+        if self.du.platform == 'craysim':
+            self.scheduler.add_resource('hbmem')
```

Comment:
@riyazhakki can you also check that it is not PTL framework which is clearing up hbmem while running setup? If so then we should fix that too (maybe not in this PR but log a bug if setup is clearing up hbmem).

Reply:
@arungrover, Yes, revert_to_defaults in setup clearing this up, we have to add code in the PTL framework to avoid this. We will fix this maybe on another PR.